composer.json

```diff
@@ -166,10 +172,12 @@
         "spryker/oauth-company-user": "^2.2.0",
         "spryker/oauth-customer-connector": "^1.8.0",
         "spryker/oauth-dummy": "^1.1.0",
+        "spryker/oms": "^11.23.1",
```

Comment:
you've added it to the composer.json 

- spryker/oms
- spryker/payment
- spryker/product-review
- spryker/product-review-gui

These dependencies are already sub-dependency of spryker-feature/sales, spryker-feature/reviews, spryker-feature/payments.

so please do `composer update package:version` not `composer require` for those packages.

Reply:
fixed